proposals/function-pointers.md

```diff
@@ -198,17 +196,17 @@ This means that it is possible to overload on `void*` and a `func*` and still se
 
 ## Open Issues
 
-### NativeCallback Attribute
+### NativeCallableAttribute
 This is an attribute used by the CLR to avoid the managed to native prologue when invoking. Methods marked by this 
 attribute are only callable from native code, not managed (can’t call methods, create a delegate, etc …). The attribute
 is not special to mscorlib; the runtime will treat any attribute with this name with the same semantics. 
 
 It's possible for the runtime and language to work together to fully support this. The language could choose to treat
-address-of `static` members with a `NativeCallback` attribute as a `func*` with the specified calling convention.
+address-of `static` members with a `NativeCallable` attribute as a `func*` with the specified calling convention.
 
 ``` csharp
-unsafe class NativeCallbackExample {
-    [NativeCallback(CallingConvention.CDecl)]
+unsafe class NativeCallableExample {
+    [NativeCallable(CallingConvention.CDecl)]
     static extern bool CloseHandle(IntPtr p);
```

Comment:
The NativeCallable methods are typically going to have a body in C#. A more realistic example may look like this:

```suggestion
    static void Close(IntPtr p) => Marshal.FreeHGlobal(p);
```

Reply:
If you go with my suggestions, `bool` needs to be changed to `void` in the following lines.